sdk/storage/azure-storage-blob/tests/test_container.py

```diff
@@ -1814,7 +1811,7 @@ def recursive_walk(prefix):
     @BlobPreparer()
     def test_list_blobs_with_include_multiple(self, storage_account_name, storage_account_key):
         bsc = BlobServiceClient(self.account_url(storage_account_name, "blob"), storage_account_key)
-        pytest.skip("Waiting on metadata XML fix in msrest")
+        # pytest.skip("Waiting on metadata XML fix in msrest")
```

Comment:
Was this intentional? Seems odd the other tests would work but this one would not if whatever "fix" was fixed.